github-metrics/write-to-db.js

```diff
@@ -0,0 +1,22 @@
+import { MongoClient } from 'mongodb';
+
+async function addMetricsToAtlas(metricsDoc) {
+    const uri =  process.env.ATLAS_CONNECTION_STRING;
+    const client = new MongoClient(uri);
+    try {
+        await client.connect();
+
+        // Currently, the only metrics we're tracking are coming from GitHub, so the DB is hard-coded here
+        // Propose we name the collection for the GitHub repo (or maybe 'owner_repo' to avoid namespace issues?)
```

Comment:
👍 I like `owner_repo`. nice and easy

Reply:
Yeah, that's how I set it up in the end. I should update this comment.